test-positive/mermaid.md

```diff
@@ -3,26 +3,15 @@
 graph TD
 A-->B("hello 🐛")
 ```
-2. flowchart1.mmd
+2. git-graph.mmd
 ```mermaid
-sequenceDiagram
-%% See https://mermaidjs.github.io/sequenceDiagram.html
-
-loop everyday D-2 working days 16:00
-    ABCD->>DEE: [14:25] BEE (Calculation per TC until ACK becomes OK)
-    
-    ABCD->>FGG: BEE (Calculation per TC until ACK becomes OK)
-    Note over ABCD,FGG: D-2 weekdays at 14:25
-    ABCD->>HII: Projected Authorisations to HII
-    Note over ABCD,HII: D-2 weekdays at 16:00
-end
-
-loop D-1 before deadline at 7:45
-    HII->>DEE: Submission
-    Note over HII,DEE: D-1 before deadline
-    HII->>FGG: Submission
-    Note over HII,FGG: D-1 before deadline
-end
```

Comment:
As you can see, this is a duplicate of the 7th diagram

https://github.com/mermaid-js/mermaid-cli/blob/8af6502057b2338f087cf7925e4d7fb4254d46b3/test-positive/mermaid.md?plain=1#L95-L115

Reply:
You are rocking!!!